### ISSUE TITLE
Advice from Bjorn

### ISSUE THREAD

**Issue by karai17** [karai17/Simple-Tiled-Implementation#11]
[17:34:00] <bjorn> Karai: Basically you should probably take the following steps:
[17:34:33] <bjorn> 1. Make sure that for each tile, there is a table with whatever information needs to be there to render it. There's only so many tiles, so this isn't a lot of data.
[17:35:00] <bjorn> 2. During loading, change each layer.data[index] from the gid it is now to a reference to one of these tile tables.
[17:36:16] <bjorn> 3. During rendering, do a double-for loop over y and x, then do local tile = layer.data[y \* width + x + 1] and render this tile at x \* tileWidth, y \* tileHeight.
[17:36:50] <bjorn> This index assumes your x/y iterate from 0 to width-1/height-1.

tl;dr build a table with y/x coord and iterate through them. should probably store the tileset index in the tile grid, too
